src/main/java/com/minecolonies/blockout/controls/ItemIcon.java

```diff
@@ -15,7 +16,7 @@
  */
 public class ItemIcon extends Pane
 {
-    private static final float GUI_ITEM_Z_LEVEL = 200.0F;
+    private static final float GUI_ITEM_Z_LEVEL     = 200.0F;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "GUI_ITEM_Z_LEVEL" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1068)